frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -792,6 +792,8 @@ global.user_groups.add(backend);
         var returned = ct.compose_content_begins_typeahead.call(
             begin_typehead_this, input
         );
+        // console.log(returned);
+        // console.log(reference);
```

Comment:
I assume this is manual testing code that's no longer needed?

Reply:
Just noticed. Fixing.